engine/engine.go

```diff
@@ -45,11 +44,11 @@ type TxValidator interface {
 
 func NewEngineAPI(b *builder.Builder, txValidator TxValidator, adapter PayloadTxAdapter, blockStore BlockStore) *EngineAPI {
 	return &EngineAPI{
-		txValidator:  txValidator,
-		blockStore:   blockStore,
-		builder:      b,
-		payloadStore: payloadstore.NewPayloadStore(),
-		adapter:      adapter,
+		txValidator: txValidator,
+		blockStore:  blockStore,
+		builder:     b,
+		payloadAttr: nil,
```

Comment:
We don't need to explicitly set the field to it's zero value.

Reply:
6fe0708